README.Rmd

```diff
@@ -62,30 +59,19 @@ library(cfr)
 # Load the Ebola 1976 data provided with the package
 data("ebola1976")
 
-# read delay distribution for ebolavirus onset to death from {epiparameter}
-# accesses parameters reported in https://doi.org/10.1016/S0140-6736(18)31387-4
-onset_to_death_ebola <- epiparameter::epidist_db(
-  disease = "Ebola Virus Disease",
-  epi_dist = "onset_to_death",
-  author = "The-Ebola-Outbreak-Epidemiology-Team",
-  single_epidist = TRUE
-)
-
 # Calculate the static CFR without correcting for delays
 cfr_static(data = ebola1976)
 
 # Calculate the static CFR while correcting for delays
 cfr_static(
   data = ebola1976,
-  epidist = onset_to_death_ebola
+  delay_density = function(x) dgamma(x, shape = 2.40, scale = 3.33)
 )
 ```
 
 ### Change in real-time estimates of overall severity during the 1976 Ebola outbreak
 
-In this example we show how the estimate of overall severity can change as more data on cases and deaths over time becomes available, using the function `cfr_rolling()`. Because there is a delay from onset-to-death, a simple "naive" calculation that just divides deaths-to-date by cases-to-date will underestimate severity. The `cfr_rolling()` function uses the `cfr_severity()` adjustment to account for delays, and instead compares deaths-to-date with cases-with-known-outcome-to-date.
```

Comment:
Why remove some of this information? I think this is quite informative to readers that are not familiar with the method.

Reply:
Re-added in https://github.com/epiverse-trace/cfr/pull/88/commits/96bcad74cf4e9dc0b1d3e4a30d06071c18485505. I felt it was too detailed for the Readme, but no strong preferences.